### PR TITLE
Add full PR template link in template

### PR DESCRIPTION
**Pull request by zombieJ**
### This is a ...

- [ ] New feature
- [x] Other

### What's the background?
  
Update RP template to add the collaborator PR template link:

### CODE REVIEW

**Comment by afc163** [comment]
Full PR template for feature request

**Comment by afc163** [comment]
Doc is ready or not need =>  Doc updated or not need

可以随便改了

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
+- [ ] Other (about what?)
 
-  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.
+### What's the background?
+  
+> 1. Describe the source of requirement.
+> 2. Resolve what problem.
+> 3. Related issue link.
+  
+### API Realization (Optional)
```
Comment (afc163) [inline .github/PULL_REQUEST_TEMPLATE.md]: Optional => Optional if not new feature

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
 
-Please makes sure that these checkboxes are checked before submitting your pull request, thank you!
+Please makes sure that these form are filled before submitting your pull request, thank you!
 
-* [ ] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
-* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
-* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
-* [ ] Rebase before creating a pull request to keep commit history clear.
-* [ ] Add some descriptions and refer relative issues for you pull request.
+[[中文版 wiki](https://github.com/ant-design/ant-design/wiki/PR-%E8%A7%84%E8%8C%83#pull-request-%E6%A8%A1%E6%9D%BF)]
```
Comment (afc163) [inline .github/PULL_REQUEST_TEMPLATE.md]: Pull Request 规范中文说明

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (afc163) [reply .github/PULL_REQUEST_TEMPLATE.md]: Realization => Implementation

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (zombieJ) [reply .github/PULL_REQUEST_TEMPLATE.md]: 我直接加链接让用户可以 preview 点击切换成中文的吧

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
+> 2. Resolve what problem.
+> 3. Related issue link.
+  
+### API Realization (Optional)
+  
+> 1. Basic thought of solution and other optional proposal.
+> 2. List final API realization and usage sample.
+> 3. GIF or snapshot should be provided if includes UI/interactive modification.
+  
+### What's the affect? (Optional)
```
Comment (afc163) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should affect be `effect`?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,22 +1,42 @@
...
+> 2. What will say in changelog?
+> 3. Does this PR contains potential break change or other risk?
 
-  * [ ] Update API docs for the component.
-  * [ ] Update/Add demo to demonstrate new feature.
-  * [ ] Update TypeScript definition for the component.
-  * [ ] Add unit tests for the feature.
+### Self Check before Merge
+
+- [ ] Doc is ready or not need
```
Comment (afc163) [inline .github/PULL_REQUEST_TEMPLATE.md]: Xxxx is updated/provided or not needed

**Comment by afc163** [comment]
然后可以自用一下 😆

**Pull request merged**